frontend/src/static/js/components/webstatus-overview-page.ts

```diff
@@ -130,16 +130,46 @@ export class OverviewPage extends LitElement {
       .then(allFeatures => {
         // Use CELL_DEFS to define the columns and
         // get the current (active) columns.
+        const columns: string[] = [];
         const columnKeys = parseColumnsSpec(getColumnsSpec(this.location));
-        const columns = columnKeys.map(
-          columnKey => CELL_DEFS[columnKey].nameInDialog
-        );
+
+        const pushBrowserName = (name: string) => {
```

Comment:
If we ever change the ColumnKey.nameInDialog for the Browser Implementation, this would pattern matching would break.

Instead, let's pass in the ColumnKey itself.

```
// This union type could be made into a union type outside of this and referred to here too.
const pushBrowserName  = (browserColumnKey: ColumnKey.StableChrome| ColumnKey.StableEdge | ColumnKey.StableFirefox | ColumnKey.StableSafari | ColumnKey.ExpChrome | ... ) => {
   let browser: string;
   let channel: string;
   if (ColumnKey.StableChrome === browserColumnKey || ColumnKey.ExpChrome === browserColumnKey) name = "Chrome";
   // do the same thing for the other browsers.
   ..
   ..
   ..
   if (browserColumnKey === ColumnKey.StableChrome || browserColumnKey === ColumnKey.StableChrome || the other stables) channel = "Stable";
   // Do the same thing for experimental
```

It could be a type

Reply:
That's kind of tedious, and more code to maintain.  How about if we just make the browser and channel explicit in the CELL_DEFS records?  I see the options field has the browser already. I will add the channel.